### PR TITLE
Make cardview heading configurable

### PR DESCRIPTION
**Pull request by suzyh**

### CODE REVIEW

File: client/cz-cardview.html [hunk 1]
```diff
@@ -152,6 +152,7 @@
       },
 
       attached: function() {
+        registerSource('cz-config', 'cardview-heading', cvHeading => this.set('cardview-heading', cvHeading));
```
Comment (shans) [inline client/cz-cardview.html]: Add a default value (it can be something boring) so configs that don't specify a heading have something.

**Comment by shans** [comment]
LGTM once default added.

**Comment by suzyh** [comment]
I think the empty string is a reasonable default, but I'm happy to change it if you actually meant a non-empty default.

**Comment by shans** [comment]
Yeah, I think something simple (even just "Card Feed") as a placeholder so people know there's something there.

**Comment by suzyh** [comment]
Done. Messed with the branch history in the process, so I've confused this pull request a little bit, but oh well...

**Pull request merged**